js/android/app/src/main/java/tech/berty/gobridge/BertyNearbyDriver.java

```diff
@@ -7,23 +7,23 @@
 import android.util.Base64;
 
 import bertybridge.Bertybridge;
-import bertybridge.NativeNBDriver;
+import bertybridge.ProximityDriver;
 
 import java.io.File;
 import java.nio.charset.StandardCharsets;
 
 import bertybridge.ProximityTransport;
-import tech.berty.gobridge.proximitydriverssdk.base.NearbyDriverSDK;
-import tech.berty.gobridge.proximitydriverssdk.lifecycle.UserAcceptCallback;
-import tech.berty.gobridge.proximitydriverssdk.lifecycle.UserConnectionCallback;
-import tech.berty.gobridge.proximitydriverssdk.lifecycle.UserMessageCallback;
-import tech.berty.gobridge.proximitydriverssdk.lifecycle.UserRequestCallback;
-import tech.berty.gobridge.proximitydriverssdk.lifecycle.UserSearchCallback;
-import tech.berty.gobridge.proximitydriverssdk.model.Endpoint;
+import tech.berty.gobridge.nearbydriversdk.base.NearbyDriverSDK;
+import tech.berty.gobridge.nearbydriversdk.lifecycle.UserAcceptCallback;
+import tech.berty.gobridge.nearbydriversdk.lifecycle.UserConnectionCallback;
+import tech.berty.gobridge.nearbydriversdk.lifecycle.UserMessageCallback;
+import tech.berty.gobridge.nearbydriversdk.lifecycle.UserRequestCallback;
+import tech.berty.gobridge.nearbydriversdk.lifecycle.UserSearchCallback;
+import tech.berty.gobridge.nearbydriversdk.model.Endpoint;
```

Comment:
Can you rename `nearbydriversdk` -> `nearbydriver` and `NearbyDriverSDK` -> `NearbyDriver`?
As it's not an SDK but actually a driver using the SDK ;)

Reply:
(Same things goes for all the imports, filenames, etc... below)